ccd-definition/CaseTypeTab/CareSupervision/8_Confidential.json

```diff
@@ -4,7 +4,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "Confidential",
-    "TabLabel": "Confidential",
+    "TabLabel": "Confidential information",
```

Comment:
is this expected change in scope of this story? I don't see anything in story/AC about it.

Reply:
Suggest by design, comment is left on the ticket.